fluentd/templates/cluster-role.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.create }}
+apiVersion: {{ include "rbac.apiVersion" . }}
+kind: ClusterRole
+metadata:
+  labels:
+    {{- include "fluentd.labels" . | nindent 4 }}
+  name: {{ include "fluentd.fullname" . }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
```

Comment:
I don't need this rule. Is there use case for collecting pods information in fluentd?

Reply:
@cw-sakamoto 
This rule is for collecting logs in `DaemonSet`.
This is used to give meta-information in [fluent-plugin-kubernetes_metadata_filter](https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter) at that time.